src/main/java/org/broadinstitute/hellbender/utils/clipping/ClippingOp.java

```diff
@@ -174,7 +172,13 @@ private GATKRead applyREVERT_SOFTCLIPPED_BASES(final GATKRead read) {
             // that cache these values, such as SAMRecord), and again after the hard clip.
             unclipped.setPosition(unclipped.getContig(), 1);
             unclipped = applyHARDCLIP_BASES(unclipped, 0, - newStart);
-            unclipped.setPosition(unclipped.getContig(), 1);
```

Comment:
Minor thing I have noticed, You should probably add comments in the method contracts warning that these methods COULD return empty/unmapped reads

Reply:
We'll do this as part of https://github.com/broadinstitute/gatk/issues/4204